patches/chrome-browser-ui-cocoa-main_menu_builder.mm.patch

```diff
@@ -11,3 +11,11 @@ index b0f66cc832e85b3d9da0b64ed1c3536563b16656..00414aa8a7edc794ba901d0ea92f44db
                  Item(IDS_REOPEN_CLOSED_TABS_MAC)
                      .command_id(IDC_RESTORE_TAB)
                      .remove_if(is_pwa),
+@@ -427,6 +429,7 @@ base::scoped_nsobject<NSMenuItem> BuildHelpMenu(
+ #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+             Item(IDS_FEEDBACK_MAC).command_id(IDC_FEEDBACK),
+ #endif
++                BRAVE_WEBCOMPAT_REPORTER_MENU_ENTRY
```

Comment:
the naming convention would be `BRAVE_BUILD_HELP_MENU`. If the naming is webcompat specific then it defeats the purpose of using a define to make it extensible